packages/authentication/test/poc/mock-express-response.js

```diff
@@ -0,0 +1,71 @@
+var http = require('http');
+var mimeTypes = require('./mime-types');
+var res = Object.create(http.ServerResponse.prototype);
+
+var exts = {};
+Object.keys(mimeTypes).forEach(function(key) {
+  var extArray = mimeTypes[key];
+  extArray.forEach(function(ext) {
+    exts[ext] = key;
+  });
+});
```

Comment:
Can we use mime-type's [mime.lookup(path)](https://www.npmjs.com/package/mime-types#mimelookuppath) instead? I believe that's what express is using under the hood too, see:

 https://github.com/expressjs/express/blob/c087a45b9cc3eb69c777e260ee880758b6e03a40/lib/response.js#L552-L559
```js

res.contentType =
res.type = function contentType(type) {
  var ct = type.indexOf('/') === -1
    ? mime.lookup(type)
    : type;

  return this.set('Content-Type', ct);
};
```

https://github.com/expressjs/express/blob/c087a45b9cc3eb69c777e260ee880758b6e03a40/lib/response.js#L30-L32
```js
var send = require('send');
var extname = path.extname;
var mime = send.mime;
```

https://www.npmjs.com/package/send#mime
```
### .mime

The `mime` export is the global instance of of the
[`mime` npm module](https://www.npmjs.com/package/mime).

This is used to configure the MIME types that are associated with file extensions
as well as other options for how to resolve the MIME type of a file (like the
default type to use for an unknown file extension).
```


Let's keep the amount of duplicated code as small as feasible.